rotkehlchen/db/upgrades/v34_v35.py

```diff
@@ -50,6 +50,10 @@ def _rename_assets_identifiers(cursor: 'DBCursor') -> None:
     cursor.executemany('UPDATE OR IGNORE assets SET identifier=? WHERE identifier=?', sqlite_tuples)  # noqa: E501
 
 
+def _add_blockchain_column_web3_nodes(cursor: 'DBCursor') -> None:
+    cursor.execute("ALTER TABLE web3_nodes ADD COLUMN blockchain TEXT NOT NULL DEFAULT 'ETH'")
```

Comment:
This should also remove the ETH default right? Also you will need to do update the currently inserted rows to add the ETH value in the blockchain column

Reply:
> Also you will need to do update the currently inserted rows to add the ETH value in the blockchain column

This is exactly what `DEFAULT 'ETH'` does, so why should it be removed?

Reply:
I got what you meant and have implemented your proposed approach